src/component/mxgraph/shape/StartEventShape.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { MxGraphFactoryService } from '../../../service/MxGraphFactoryService';
+import { mxgraph } from 'ts-mxgraph';
+
+const mxEllipse: typeof mxgraph.mxEllipse = MxGraphFactoryService.getMxGraphProperty('mxEllipse');
+
+export default class StartEventShape extends mxEllipse {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public constructor(bounds: mxgraph.mxRectangle, fill: any, stroke: any, strokewidth = 2) {
```

Comment:
I am thinking that maybe it is a good idea to have our constants defined somewhere in one place?
DrawingConstants {
STROKE_WIDTH_NORMAL = 2
STROKE_WIDTH_BOLD = 4
...
}

Reply:
Done